cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/ProgramScheduleStoreDataset.java

```diff
@@ -481,6 +501,10 @@ public ProgramScheduleRecord getScheduleRecord(ScheduleId scheduleId) throws Not
 
   /*------------------- private helpers ---------------------*/
 
+  /**
+   * List schedules in a given application and optionally filtered by the program triggered by the schedule
```

Comment:
also the comment is not super clear.. this method gets the schedules in the application and if program id is provided it will only get schedules associated with that program right? we should not say `triggered` then..right?